src/chart/generator/marker.cpp

```diff
@@ -166,8 +165,15 @@ void Marker::setIdOffset(size_t offset)
 
 std::string Marker::toJSON() const
 {
-	std::string res;
-	Conv::JSONObj{res}("categories",
+	static std::string res;
+	Conv::JSONObj jsonObj{res};
```

Comment:
Inline it. Make appendToJSON accept JSONObj&&

///
Create a scope for it if it's not will be inline